data/saas/config/mixpanel_config.yml

```diff
@@ -0,0 +1,99 @@
+saas_config:
+  fides_key: mixpanel_connector_example
+  name: Mixpanel SaaS Config
+  description: A sample schema representing the Mixpanel connector for Fidesops
+  version: 0.0.1
+
+  connector_params:
+    - name: api_secret
+    - name: token
+    - name: gdpr_oauth_token
+    # TODO - OPTION wasn’t supported at the time it was written.
```

Comment:
Since the `us_or_eu` paramSpec is just used to determine which domain to use, we can just use the `domain` directly instead for now.

Reply:
Written as domain only

Reply:
It is implemented as such